.bazelignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
I think you want REPO.bazel instead since it gives you recursive globs, see https://github.com/aspect-build/aspect-workflows-template/blob/main/%7B%7B%20.ProjectSnake%20%7D%7D/REPO.bazel